python/triton/runtime/build.py

```diff
@@ -63,13 +64,19 @@ def _build(name, src, srcdir, library_dirs, include_dirs, libraries):
         cc_cmd = [cxx]
         if icpx is not None:
             cc_cmd += ["-fsycl"]
+        else:
+            cc_cmd += ["--std=gnu++17"]
     else:
         cc_cmd = [cc, "-O3"]
 
     cc_cmd += [src, "-shared", "-fPIC", "-o", so]
     cc_cmd += [f'-l{lib}' for lib in libraries]
     cc_cmd += [f"-L{dir}" for dir in library_dirs]
     cc_cmd += [f"-I{dir}" for dir in include_dirs if dir is not None]
+
+    if os.getenv('VERBOSE') is not None:
```

Comment:
What about `TRITON_XPU_VERBOSE` instead of `VERBOSE`?

Also
```suggestion
    if os.getenv('VERBOSE'):
```

Reply:
I'm glad with any name. Just `VERBOSE` is used in most build tools to display compile command line

Reply:
My concern is that this branch can be used not only with XPU.